ddl/table.go

```diff
@@ -335,6 +341,34 @@ func splitTableRegion(store kv.Storage, tableID int64) {
 	}
 }
 
+func preSplitTableRegion(store kv.Storage, tblInfo *model.TableInfo) {
+	s, ok := store.(splitableStore)
+	if !ok {
+		return
+	}
+	// Split table region.
+	step := int64(1 << (tblInfo.ShardRowIDBits - tblInfo.PreSplitRegions))
```

Comment:
The `PreSplitRegions` means the number of prefix bits used for splitting? the name confused me.

Reply:
PreSplitRegions means will pre-split 2^(PreSplitRegions - 1) regions.

Reply:
The name is very confusing. maybe name it `PreSplitByShardbits`

Reply:
ok.